frontend/views/containers/modals/DesignSystem.vue

```diff
@@ -24,7 +33,8 @@ modal-template(:class="{ 'has-background': background }")
       ) Toggle background
 
   template(#footer='')
-    i18n(tag='p') According to your voting rules, 8 out of 10 members will have to agree with this.
+    //- We aren't using i18n because this string shouldn't be translated either
```

Comment:
I'm fine with it, don't forget to remove these comments before merging the PR.

Reply:
Why remove the comments? They're important, they're for anyone who wants to use this file as an example, to remind them to change these lines...